nifi-pulsar-processors/src/main/java/org/apache/nifi/processors/pulsar/AbstractPulsarConsumerProcessor.java

```diff
@@ -76,6 +70,11 @@
     static final AllowableValue FAIL = new AllowableValue(ConsumerCryptoFailureAction.FAIL.name(), "Fail",
             "Report a failure condition, and then route the message contents to the FAILED relationship.");
 
+    static final AllowableValue OFFSET_EARLIEST = new AllowableValue("Earliest", "Earliest",
+            "Set the offset to the earliest position");
```

Comment:
Please change the descriptions to match the enum definitions found in the API [docs](https://pulsar.apache.org/api/client/2.9.0-SNAPSHOT/org/apache/pulsar/client/api/SubscriptionInitialPosition.html#Earliest), 

E.g. "The earliest position which means the start consuming position will be the first message."  Just to clarify this to the NiFi users.  Thanks